assets/src/lightbox/web-stories-lightbox.js

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Handles the 'amp-story-player' lightbox.
+ *
+ * Displays the story that user interacted with in a lightbox.
+ */
+class Lightbox {
```

Comment:
Is there a reason we are using a class here? We are using functional programming everywhere else.

Reply:
I think we've discussed this before. For this use case it seems quite fitting and easier to read & maintain this way.